ginko/src/dts/project.rs

```diff
@@ -74,11 +74,24 @@ impl ProjectFile {
 #[derive(Default)]
 pub struct Project {
     files: HashMap<PathBuf, ProjectFile>,
+    pub include_paths: Vec<PathBuf>,
     pub severities: SeverityMap,
 }
 
 impl Project {
-    pub fn add_file(&mut self, file_name: PathBuf) -> Result<(), io::Error> {
+    pub fn set_include_paths(
```

Comment:
I'm a bit undecided about this throwing an error if the path doesn't exist.
On the one hand I like it as it's transparent to the user that the input they provided might be faulty. On the other hand, I think that there are use-cases where one would simply want to add a lot of include paths by default (i.e., in a project and team-wide configuration), so that some file can be found on any machine.
Could you elaborate a bit on your choice of requiring the include paths to exist?

Reply:
No strong opinion on that, I thought the behavior should match `add_file` but it will also later on be covered by the `Include` path resolver.